scionlab/models/vpn.py

```diff
@@ -105,6 +99,47 @@ def _find_client_ip(self):
                 return ip
         raise RuntimeError('No free client IP available')
 
+    def clean(self):
+        try:
+            subnet = ipaddress.ip_network(self.subnet)
+        except ValueError:
+            raise ValidationError('Invalid subnet', code='vpn_bad_subnet')
+        try:
+            server_vpn_ip = ipaddress.ip_address(self.server_vpn_ip)
+        except ValueError:
+            raise ValidationError('Invalid server VPN IP address', code='vpn_bad_server_vpn_ip')
+        if server_vpn_ip not in subnet:
+            raise ValidationError('Server VPN IP is not inside the specified subnet',
+                                  code='server_vpn_ip_not_in_subnet')
+
+    def save(self, **kwargs):
+        self.clean()
+        super().save(**kwargs)
+
+    def update(self,
+               server=_placeholder,
+               server_port=_placeholder,
+               subnet=_placeholder,
+               server_vpn_ip=_placeholder,
+               private_key=_placeholder,
+               cert=_placeholder):
+        """
+        Updates all/any parameter. It always saves the object even when no changes
+        """
+        if server is not _placeholder:
+            self.server = server
+        if server_port is not _placeholder:
+            self.server_port = server_port
+        if subnet is not _placeholder:
+            self.subnet = subnet
+        if server_vpn_ip is not _placeholder:
+            self.server_vpn_ip = server_vpn_ip
+        if private_key is not _placeholder:
+            self.private_key = private_key
+        if cert is not _placeholder:
+            self.cert = cert
+        self.save()
```

Comment:
Should actually bump the version on the `server` as well as the previous server if it's changed.